src/lib/components/comparisonTable/components/TableInfoButton/style.module.scss

```diff
@@ -25,7 +25,7 @@
   }
 
   svg {
-    fill: $ds-primary-100;
+    fill: $ds-grey-400;
```

Comment:
Not necessary as part of this PR but if you want, you can also replace this inline SVG with
`<InfoInfo color="grey-400" size={20 /* or other */} />

Reply:
Sounds nifty, but how would I then give it hover states and similar? I tried manipulating it with

```
svg {
 fill: SOMECOLOR
}
```

to no avail :/
Looks like the fill is set to "currentColor" at all times

Reply:
Nvm, I need to use svg -> path instead.

Reply:
You can then remove the svg fill color as the Icon component captures the button color (that's what currentColor is doing):
```
.button {
  color: $ds-primary-400;
  
  &:hover {   
    color: $ds-primary-300;
  }
}
```

Reply:
Ah cool 👍 Will keep that in mind and use it in the future! Here however, for better a11y, I wrapped the icon in a real button and just tweaked the original custom css instead of creating 2 new classes (one for the wrapping button and one for the icon itself).